### PR TITLE
Log rectification

### PR DESCRIPTION
**Pull request by zdj6373**
Log error, modify

### CODE REVIEW

**Comment by zdj6373** [comment]
@k8s-bot ok to test

File: pkg/controller/controller_ref_manager.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func GetControllerOf(controllee *v1.ObjectMeta) *metav1.OwnerReference {
 func (m *PodControllerRefManager) AdoptPod(pod *v1.Pod) error {
 	// we should not adopt any pods if the controller is about to be deleted
 	if m.controllerObject.DeletionTimestamp != nil {
-		return fmt.Errorf("cancel the adopt attempt for pod %s because the controlller is being deleted",
+		return fmt.Errorf("Cancel the adopt attempt for pod %s because the controller is being deleted",
```
Comment (derekwaynecarr) [inline pkg/controller/controller_ref_manager.go]: by convention, we tend to have all errors start with lower case, so "cancel" not "Cancel"

**Review by derekwaynecarr — DISMISSED** [review]
please make the one update, then this is fine.

**Comment by derekwaynecarr** [comment]
@k8s-bot ok to test

File: pkg/controller/controller_ref_manager.go (hunk 1 above)
Comment (zdj6373) [inline pkg/controller/controller_ref_manager.go]: @derekwaynecarr   Already modified，THKS

**Comment by mikedanese** [comment]
@k8s-bot kops aws e2e test this

**Comment by mikedanese** [comment]
@k8s-bot gci gce e2e test this

**Pull request merged**